docs/ref/request-response.txt

```diff
@@ -735,16 +736,15 @@ Attributes
 Methods
 -------
 
-.. method:: HttpResponse.__init__(content='', content_type=None, status=200, reason=None, charset=None)
+.. method:: HttpResponse.__init__(content=b'', content_type=None, status=200, reason=None, charset=None)
 
     Instantiates an ``HttpResponse`` object with the given page content and
     content type.
 
-    ``content`` should be an iterator or a string. If it's an
-    iterator, it should return strings, and those strings will be
-    joined together to form the content of the response. If it is not
-    an iterator or a string, it will be converted to a string when
-    accessed.
+    ``content`` should be an iterator, bytestring, or a string. If it's an
+    iterator, it should return strings or bytestrings, and those will be
+    joined together to form the content of the response. Other types will be
+    converted to a bytestring when accessed.
```

Comment:
I'm not sure if it's clear what "Other types will be converted to a bytestring when accessed." means. It might be helpful to me more explicit about that.